lib/boom_notifier.ex

```diff
@@ -16,11 +16,11 @@ defmodule BoomNotifier do
         callback.(settings[:notifier], settings[:options])
 
       [missing_key] ->
-        Logger.error("Settings error: #{inspect(missing_key)} parameter missing")
+        Logger.error("Boom Notifier Router error: #{inspect(missing_key)} parameter missing")
```

Comment:
What do you think about doing something like this?

```suggestion
        Logger.error("(BoomNotifier) #{inspect(missing_key)} parameter is missing")
```

That way it looks more similar to this
```
(Mix) Could not start application my_app
```